### PR TITLE
[MM-48798] - Typo in pricing modal

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Fix typo in pricing modal from `Dedicated encryption keys 99%` to `Dedicated encryption keys`

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-48798

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by emmyni — APPROVED** [review]

File: i18n/en_AU.json [hunk 1]
```diff
@@ -4348,7 +4348,7 @@
   "pricing_modal.addons.USSupport": "United States-only based support",
   "pricing_modal.addons.dedicatedDB": "Dedicated database",
   "pricing_modal.addons.dedicatedDeployment": "Dedicated virtual secure cloud deployment (Cloud)",
-  "pricing_modal.addons.dedicatedEncryption": "Dedicated encryption keys 99%",
+  "pricing_modal.addons.dedicatedEncryption": "Dedicated encryption keys",
```
Comment (neallred) [inline i18n/en_AU.json:4351]: Even though we likely know what this should be, we should only change `en.json`. IIRC trying to change other languages/dialects we know can confuse the downstream translator process.

**Review by neallred — APPROVED** [review]
LGTM, but let's revert the `en_AU.json` change.

File: i18n/en_AU.json (hunk 1 above)
Comment (AGMETEOR) [reply i18n/en_AU.json:4351]: Alright. Reverting...

**Review by yasserfaraazkhan — APPROVED** [review]
changes are reflecting 

Screenshot 2022-12-05 at 7 19 31 PM

**Pull request merged**